### PR TITLE
map volume "GET /" to uiStatusHandler for convenience

### PR DESCRIPTION
**Pull request by bingoohuang**
1. map `GET /` with uiStatusHandler same to `GET /ui/index.html` for convenience.
2. refactor some local variables name from under_scored to camelCase

### CODE REVIEW

**Comment by chrislusf** [comment]
avoid these shortcuts to make enforcing access control easier.

File: weed/server/volume_server_handlers.go [hunk 1]
```diff
@@ -22,6 +22,11 @@ security settings:
 */
 
 func (vs *VolumeServer) privateStoreHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method == "GET" && r.URL.Path == "/" { // for convenience to /ui/index.html
```
Comment (chrislusf) [inline weed/server/volume_server_handlers.go:25]: do not do this. not easy to manage access.

Comment (bingoohuang) [reply weed/server/volume_server_handlers.go:25]: Got it.

**Comment by chrislusf** [comment]
the other refactoring/reformatting is good. you can send separate PR for this. Try not to mix the refactoring with the actual change.

**Pull request closed without merging**